core/templates/components/state-editor/state-solution-editor/state-solution-editor.component.ts

```diff
@@ -131,8 +131,17 @@ export class StateSolutionEditorComponent implements OnInit {
 
   getSolutionSummary(): string {
     let solution = this.stateSolutionService.savedMemento;
-    let solutionAsPlainText = (
-      solution.getSummary(this.stateInteractionIdService.savedMemento));
+    let interactionId = this.stateInteractionIdService.savedMemento;
+    let solutionAsPlainText = '';
+    if (interactionId === 'DragAndDropSortInput') {
+      solutionAsPlainText = (
+        solution.getSummary(
+          interactionId, this.stateCustomizationArgsService.savedMemento
+        ));
+    } else {
+      solutionAsPlainText = (
+        solution.getSummary(this.stateInteractionIdService.savedMemento));
+    }
     solutionAsPlainText = (
       this.convertToPlainText.transform(solutionAsPlainText));
```

Comment:
Why reassign this variable? It seems like solutionAsPlainText does not refer to the right thing when it is initialised.

Reply:
I've modified it and added a variable with a different name, PTAL